test/repository_test.rb

```diff
@@ -68,6 +72,17 @@
             UserRepository.persist(persisted_user)
             persisted_user.created_at.must_equal persisted_user.created_at
           end
+
+          it 'touches updated_at' do
+            update_user = UserRepository.find(user.id)
+            update_user.name = 'My'
+
+            updated_user = UserRepository.persist(update_user)
+
+            updated_user = UserRepository.find(user.id)
+            updated_user.updated_at.must_equal updated_user.updated_at
```

Comment:
what do you think about removing this assertion because it does not really test anything